docs/icr-images.md

```diff
@@ -29,7 +29,7 @@ ibmcloud cr region-set global
 ibmcloud cr images --restrict appcafe/open-liberty
```

Comment:
I noticed there is still some reference to `icr.io/appcafe/open-liberty:24.0.0.3-kernel-slim-java17-openj9-ubi` in the example above, was that supposed to be updated?

Reply:
That's intentional. The long-supported quarterly release is referenced in the example.